util/container/README.md

```diff
@@ -0,0 +1,10 @@
+# HWPE Stream Dockertest Container
+The dockerfile contains the build instructions for the container. It is compatible for cocotb and Verilator. Once built simply go into the `/cocotb` directory and run `pytest`.
+
+## Alternative docker download
+You can download an uploaded build by running:
+``` bash
+docker pull rgantonio/cocotb-hwpe-stream
```

Comment:
This command does not work for me!

Reply:
@JosseVanDelm really? haha like it won't pull it? I tried it on a different computer and it worked... well I used podman but should be the same right? Does it say something?

Reply:
Now it seems to work :shrug: